tests/ChatTests/Direct.hs

```diff
@@ -1113,6 +1113,19 @@ testMaintenanceModeWithSettings tmp = do
       alice ##> "/_stop"
       alice <## "chat stopped"
       let settings = T.unpack . safeDecodeUtf8 . LB.toStrict $ J.encode defaultAppSettings
+      -- app-provided defaults
+      alice ##> ("/_get app settings " <> settings)
+      alice <## ("app settings: " <> settings)
+      -- parser defaults fallback
+      alice ##> "/_get app settings"
+      alice <## ("app settings: " <> settings)
```

Comment:
this doesn't test that providing app settings doesn't return default settings, first test should pass modified default settings to api